LOVE 1034 - Make more basic aws/basic-eks-cluster (#81)
xebialabs/blueprints: aws/basic-eks-cluster/README.md

> # Basic AWS Kubernetes Engine Cluster

Reviewer: ```suggestion
# Basic AWS Elastic Kubernetes Service(EKS) Cluster
```

## Introduction

Amazon Elastic Container Service for Kubernetes (EKS) allows you to deploy, manage, and scale containerized applications in the cloud using Kubernetes.

> Use this blueprint to provision a GKE cluster using CloudFormation.

Reviewer: ```suggestion
Use this blueprint to provision an EKS cluster using CloudFormation.
```
